src/ios/CDVFileTransfer.m

```diff
@@ -552,6 +552,8 @@ - (void)connectionDidFinishLoading:(NSURLConnection*)connection
 
     if (self.direction == CDV_TRANSFER_UPLOAD) {
         uploadResponse = [[NSString alloc] initWithData:self.responseData encoding:NSUTF8StringEncoding];
+        if (uploadResponse == nil)
+            uploadResponse = [[NSString alloc] initWithData: self.responseData encoding:NSISOLatin1StringEncoding];
```

Comment:
I've written some test code, but I don't have the environment to run it. Could you please give me some information about how to run those test cases? @nikhilkh @daserge